docs/Configuration/Dialplan/Pattern-Matching.md

```diff
@@ -3,331 +3,255 @@ title: Pattern Matching
 pageid: 4817353
 ---
 
-The next concept we'll cover is called *pattern matching*. Pattern matching allows us to create extension patterns in our dialplan that match more than one possible dialed number. Pattern matching saves us from having to create an extension in the dialplan for every possible number that might be dialed.
+The next concept we'll cover is called *pattern matching*. Pattern matching allows us to create extension patterns in our dialplan that match more than one possible dialed number. It saves us from having to create an extension in the dialplan for every possible number that might be dialed.
 
 When Alice dials a number on her phone, Asterisk first looks for an extension (in the context specified by the channel driver configuration) that matches exactly what Alice dialed. If there's no exact match, Asterisk then looks for a pattern that matches. After we show the syntax and some basic examples of pattern matching, we'll explain how Asterisk finds the best match if there are two or more patterns which match the dialed number.
 
-Special Characters Used in Pattern Matching
-===========================================
+**Pattern matches always begin with an underscore**. This is how Asterisk recognizes that the extension is a pattern and not just an extension with a funny name. Within the pattern, we use various letters and characters to represent a *family* of dial numbers.
```

Comment:
(optional) Family might come with some semantic connotations. I think 'set' or 'pattern' would be more neutral here.

Reply:
changed to "collection". thank you!